package.json

```diff
@@ -167,7 +167,7 @@
   },
   "lint-staged": {
     "*.js": [
-      "npm run prettier --write",
+      "prettier --write",
```

Comment:
@hudochenkov I changed this as the original command gave me a "script not found" error. Does this seem right to you?

Reply:
Yes, `prettier --write` is correct.

Reply:
Thanks 👍